src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,9 +50,17 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            /*if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
+            }*/
+            for (String str1 : wordsInName) {
+                for (String str2 : keywords) {
```

Comment:
Nested for-loops should be ignored wherever possible. Try to find ways to efficiently implement this using the existing Collections.disjoint method.